### PR TITLE
Add link to Kubernetes faster pull request review article.

### PR DESCRIPTION
**Pull request by daspecster**
Add link to article @dhermes mentioned about streamlining pull requests.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -487,3 +487,13 @@ Before we can accept your pull requests you'll need to sign a Contributor Licens
 - **If you work for a company that wants to allow you to contribute your work**, then you'll need to sign a `corporate CLA <https://developers.google.com/open-source/cla/corporate>`__.
 
 You can sign these electronically (just scroll to the bottom). After that, we'll be able to accept your pull requests.
+
+*******************************
+Helpful Pull Request Guidelines
+*******************************
```
Comment (dhermes) [inline CONTRIBUTING.rst]: Shouldn't this just be the first bullet in the `Contributing` section at the top?

Comment (dhermes) [inline CONTRIBUTING.rst]: Maybe in the "Send a pull request" bullet?

Comment (daspecster) [inline CONTRIBUTING.rst]:
```rst
############
Contributing
############

#. _Faster Pull Request Reviews
#. **Please sign one of the contributor license agreements below.**
#. Fork the repo, develop and test your code changes, add docs.
#. Make sure that your commit messages clearly describe the changes.
#. Send a pull request (See: _Faster Pull Request Reviews).
```

Comment (dhermes) [inline CONTRIBUTING.rst]: Right. Though maybe "please, pretty please, pretty pretty please read". Also that isn't the correct syntax for an RST link, though you can get that right in the code?

Comment (daspecster) [inline CONTRIBUTING.rst]: https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2784/commits/a7e136e1e13450a182508eb8e276849b12c05e31

**Comment by daspecster** [comment]
squashed, and cancelled Travis.

**Pull request merged**